### ISSUE TITLE
WARNING: Duplicate filename found

### ISSUE THREAD

**Issue by andre2007** [jenkinsci/code-coverage-api-plugin#163]
While publishing the coverage report,
```
publishCoverage adapters: [dListingAdapter(mergeToOneReport: true, path: 'source-*.lst')], sourceFileResolver: sourceFiles('STORE_LAST_BUILD')
```
I get following output in Jenkins:
```
Publishing Coverage report....
A total of 1 reports were found
Source File Navigation is enabled - Current level: STORE_LAST_BUILD
15 source files need to be copied.
WARNING: Duplicate filename found: dub.json
WARNING: Duplicate filename found: dub.json
WARNING: Duplicate filename found: dub.json
WARNING: Duplicate filename found: app.d
WARNING: Duplicate filename found: dub.json
WARNING: Duplicate filename found: dub.json
WARNING: Duplicate filename found: app.d
WARNING: Duplicate filename found: dub.json
WARNING: Duplicate filename found: app.d
WARNING: Duplicate filename found: test.d
WARNING: Duplicate filename found: hash.d
WARNING: Duplicate filename found: test.d
WARNING: Duplicate filename found: test.d
WARNING: Duplicate filename found: bcrypt.d
WARNING: Duplicate filename found: test.d
WARNING: Duplicate filename found: test.d
WARNING: Duplicate filename found: test.d
WARNING: Duplicate filename found: test.d
```

My understanding is, there are a lot of files with same file name in different folders. But why is this an issue for the code coverage plugin? The output, rendering / navigating through the findings is working fine. The warnings just makes a lot of noise in the log files.

Code Coverage API: 1.1.4
D Listing Coverage Plugin: 1.0

Could you may add an option `Hide warnings` or maybe the warnings aren't warnings, as everything seems to work fine?

**Comment by trash-80** [comment]
Andre,
They've fixed this in 1.1.5. The warnings were removed since they really didn't help.  We had the same issue as you and we ended up spending a lot of time tracing this issue down only to find out it's just an unnecessary warning.  But, that line of code was removed in the 1.1.6 release.

**Comment by andre2007** [comment]
@trash-80 perfekt. Thanks a lot for the info.
